.eslintrc.js

```diff
@@ -32,6 +31,7 @@ module.exports = {
     '@next/next/no-html-link-for-pages': ['off'],
     '@next/next/no-img-element': ['off'],
     'linebreak-style': ['error', os.platform() === 'win32' ? 'windows' : 'unix'],
+    'import/extensions': ['warn'],
```

Comment:
there shouldn't be any changes in this file

Reply:
reverted to initial state